README.md

```diff
@@ -110,7 +110,109 @@ The last published version of the package become [available for download](htt
 
 ### Examples
 
-> **Will come soon**
+Below are spome examples of JavaScript IPFS in action.
+
+#### Startup (in Node.js or browser)
+
+There's still a bit of work required to start up a node in a robust way (i.e. without requiring the user to manually `js-ipfs init`). Here is one way to do it.
+
+```javascript
+// Find the IPFS implementation
+const IPFS = typeof Ipfs !== 'undefined' ? Ipfs  // browser global
+  : typeof require === 'function' ? require('ipfs') // cjs require
+  : throw new Error('could not load IPFS')
+
+// Make an IPFS node
+var ipfs = new IPFS()
+
+// Init a repo in the given IPFS node it if hasn't got one already. Calls the
+// setup callback, passing the normal callback, after first initialization.
+// Calls the normal callback directly after subsequent initializations. Calls
+// the normal callback with an error parameter if there is an error.
+function initIfNeeded (ipfs, setup, callback) {
+  ipfs.init((err) => {
+    if (!err) {
+      // This is the first time we have started a node
+      setup(callback)
+    } else if (err.message == 'repo already exists') {
+      // The node already exists
+      callback()
+    } else {
+      callback(err)
+    }
+  })
+}
+
+// Init the node
+initIfNeeded(ipfs, (callback) => {
+  // On first initialization, do some setup
+  // Get the node config we just init-ed
+  ipfs.config.get((err, config) => {
+    if (err) {
+      throw err
+    }
+    // Add at least one libp2p-webrtc-star address. Without an address like this
+    // the libp2p-webrtc-star transport won't be installed, and the resulting
+    // node won't be able to dial out to libp2p-webrtc-star addresses.
+    var star_addr = ('/libp2p-webrtc-star/ip4/127.0.0.1/tcp/9090/ws/ipfs/' +
+      config.Identity.PeerID)
+    ipfs.config.set('Addresses.Swarm[1]', star_addr, (err) => {
+      if (err) {
+        throw err
+      }
```

Comment:
This also requires the signalling server of libp2p-webrtc-star to be up. Which makes me think that maybe we should have one always running for people to do demos that goes with the default config.

@lgierth how does this sound?

We have two options:

a) If we get a signalling server always up, then we should make WebSockets and webrtc-start be the default config values for a browser node (the same way we have the TCP Sockets)

b) If we don't get the signalling server always up. We should set the default addrs to be just websockets for browser and have a separate example that shows how to turn on WebRTC.